pkg/services/ngalert/models/alert_rule_test.go

```diff
@@ -319,33 +319,11 @@ func TestDiff(t *testing.T) {
 		if rule1.DashboardUID != rule2.DashboardUID {
 			diff := diffs.GetDiffsForField("DashboardUID")
 			assert.Len(t, diff, 1)
-
-			if rule1.DashboardUID == nil {
-				assert.True(t, diff[0].Left.IsNil())
-			} else {
-				assert.Equal(t, *rule1.DashboardUID, diff[0].Left.Elem().String())
-			}
-			if rule2.DashboardUID == nil {
-				assert.True(t, diff[0].Right.IsNil())
-			} else {
-				assert.Equal(t, *rule2.DashboardUID, diff[0].Right.Elem().String())
-			}
```

Comment:
I decided to remove these asserts because I can't figure out the behavior. Sometimes value is a pointer, sometimes not.